kettle/Makefile

```diff
@@ -31,8 +31,8 @@ build:
 	echo Built $(IMG):$(TAG) and tagged with $(IMG):latest ;
 
 push: build
-	docker push $(IMG):$(TAG)
-	docker push $(IMG):latest
+	gcloud docker -- push $(IMG):$(TAG)
```

Comment:
This is deprecated and we should very much stop using it. *docker* is no longer supporting the method this used.

Reply:
What's the replacement?

Reply:
just `docker push`, and a one-time-per-machine only `gcloud auth configure-docker`.

we do have a few things intentionally using `gcloud docker ...` (push-build.sh) but only because the auth command had a bug that caused it to not function if the docker daemon wasn't running (should be patched in the latest release)